### DISCUSSION TITLE
How do I set the VirtualizeOptions?

### DISCUSSION THREAD

**Discussion by haydebxd** [Megabit/Blazorise#4011] [Q&A]
I want to change the height of my datagrid, to fill some more of my page out, I want to set it to 800px instead of the 500px but I cannot figure out how to use Virtualizeoptions to set the DataGridHeight.

Any help would be greatly appreciated, thank you.

**Comment by stsrki** [comment]
Have you tried with `FixedHeaderDataGridHeight` and `FixedHeaderDataGridMaxHeight` on DataGrid?

**Reply by David-Moreira** [reply]
Or do you not understand how to set the parameter? You have to new up the options sometjing like : virtualize options="@new() { DataGridHeight = "500px" }"

**Reply by haydebxd** [reply]
@stsrki I have tried setting that, however I've noticed that the only thing those parameters have an effect on are keeping the datagrid height consistent. For example the grid stays the same size whether i have 50 results or 2 results. (Not what I want). I would have to upload a picture to show what is happening because it is not a desired behavior.

@David-Moreira I do not understand how to set the parameters, and that is what I was trying to ask. I will give your solution a try, thank you.
